conbench/entities/history.py

```diff
@@ -32,7 +34,14 @@ class HistorySerializer:
     many = _Serializer(many=True)
 
 
-def get_history(case_id, context_id, hardware_hash):
+def get_history(case_id, context_id, hardware_hash, repo) -> List[tuple]:
+    """Given a case/context/hardware/repo, return all non-errored BenchmarkResults
+    (past, present, and future) on the default branch that match those criteria, along
+    with information about the stats of the distribution as of each BenchmarkResult.
+    Order is not guaranteed.
+
+    Primarily used to power the blue line in the timeseries plots.
+    """
```

Comment:
Thank you for the docstring! I think (?) this also powers the `/history/` API as well, which is a rational place for anybody doing their own data investigations into their metrics to start.

Reply:
Hmm, so this _only_ gets results on the default branch. That makes sense for the blue line, but it would be very handy to have a version of this that would return `ancestor_commit_query` joined with results

Reply:
I was assuming no one except the webapp used `/api/history` so thanks for proving me wrong 🙂 maybe you could make another issue (related to #384?) that adds a way to return the non-default commits too? I'm sticking with default branch here since that was the behavior before